packages/design-system/src/components/OcDrop/OcDrop.vue

```diff
@@ -45,7 +45,8 @@ export default defineComponent({
      */
     popperOptions: {
       type: Object,
-      required: false
+      required: false,
+      default: () => {}
```

Comment:
I'd prefer if we would return an empty object here instead of an empty method.

```suggestion
      default: () => ({})
```

Reply:
Got it, this is in few other components. Gonna fix it